epimodel/utils.py

```diff
@@ -1,11 +1,14 @@
 import datetime
 import logging
 import re
+from typing import Union, Set, Optional
 
 import dateutil
 import pandas as pd
 import unidecode
 
+import epimodel
```

Comment:
This is a surprising import :grin:

Reply:
:sweat_smile:

Reply:
@hnykda Actually, not sure how to handle this better: RegionDataset is used in a type hint, but I can't import it, as `regions.py` imports `utils.py`. So I wanted to import the parent module, but I don't know how to do that as a relative import, since it is the root module .. 
What do you think?

Reply:
But you are using forward references via string anyway, no? So `rds: "epimodel.RegionDataset"` will work regardless it's imported via `epimodel`.

Another solution is to add (https://docs.python.org/3.7/whatsnew/3.7.html#pep-563-postponed-evaluation-of-annotations):
```
from __future__ import annotations
```
at the top of the `utils.py` and then use non string variant:
```
rds: epimodel.RegionDataset
```